### PR TITLE
add coffee-rails for deployment dependency - teaspoon

### PR DESCRIPTION
**Pull request by mejackreed**
Needed for deployment to dev environment

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -31,6 +31,9 @@ group :development, :test do
 
   # Have jQuery local for testing
   gem 'jquery-rails'
+
+  # Needed for deploying to dev/test
+  gem 'coffee-rails'
```
Comment (jkeck) [inline Gemfile]: Really?  That's unfortunate if we're not actually using it. :frowning:

Rails--

Comment (mejackreed) [inline Gemfile]: #teaspoon

Comment (jkeck) [inline Gemfile]: Ahhhh, that makes sense why it wasn't needed before.  I redact my criticism of rails.  Rails is Life. Rails is Love.

Comment (mejackreed) [inline Gemfile]: :tropical_drink: keep drinking the KoolAid

Comment (jkeck) [inline Gemfile]: Oh yeah!

**Pull request merged**